### DISCUSSION TITLE
Upstream Issues with Credentials Provider Error Handling

### DISCUSSION THREAD

**Discussion by aaronksaunders** [Hebilicious/authjs-nuxt#111] [General]
There are. upstream issues that make this plugin unusable for email / password sign-in, do you have any suggested workarounds?

https://github.com/nextauthjs/next-auth/issues/8340

it doesn't properly pass errors from the `authenticate` method.

**Comment by Hebilicious** [comment]
Besides fixing the upstream issue or using redirect:true, I'm not sure what you could do. There might be some workaround by manually checking that the cookies are set on the client and not relying on the signin Response.
